tools/go-agent/instrument/plugins/instrument.go

```diff
@@ -107,9 +108,8 @@ func (i *Instrument) CouldHandle(opts *api.CompileOptions) bool {
 		// check the version of the framework could handler
 		version, err := i.tryToFindThePluginVersion(opts, ins)
 		if err != nil {
-			// Local package (e.g. replaced toolkit) does not have version.
-			// So when the version is not detected, we should not skip it.
-			logrus.Warnf("the plugin %s %s", ins.Name(), err)
+			logrus.Warnf("ignore the plugin %s, because %s", ins.Name(), err)
+			continue
```

Comment:
I remember this is used to handle the check for missing version information when `replacing` in `go.mod`, @Alipebt could you help to re-check?

Reply:
> I remember this is used to handle the check for missing version information when `replacing` in `go.mod`, @Alipebt could you help to re-check?

I found the solution is not correct. Let me find another way

Reply:
If we skip all args in the loop, shall we return an error?

Reply:
I think a warm log is enough, an error will let the compile phase failure.

Reply:
> I think a warm log is enough, an error will let the compile phase failure.

![image](https://github.com/apache/skywalking-go/assets/2568208/64d886c2-6558-4fde-9572-06e811150dc9)

I meant here. The only possbility to reach here is no go sources.

Reply:
No, we don't need to add the error here. If there is no go file, then it will not have any enhancements. If an error is returned here, it will cause useless error log output in the upper layer.